doc/resourcemanagementbatch.rst

```diff
@@ -1,27 +1,32 @@
 Batch Management
 ================
 
-For general information on resource management, see :doc:`Resource Management<resourcemanagement>`.
+For more information on the Azure Batch service, check out the `Batch Documentation <https://azure.microsoft.com/en-us/documentation/services/batch/>`__.
+For working samples, see the `Batch samples repo <https://github.com/Azure/azure-batch-samples/tree/master/Python>`__.
 
-Create the management client
-----------------------------
+Create the Batch Management client
+----------------------------------
 
 The following code creates an instance of the management client.
-
 You will need to provide your ``subscription_id`` which can be retrieved
 from `your subscription list <https://manage.windowsazure.com/#Workspaces/AdminTasks/SubscriptionMapping>`__.
 
 See :doc:`Resource Management Authentication <resourcemanagementauthentication>`
-for details on getting a ``Credentials`` instance.
+for details on handling Azure Active Directory authentication with the Python SDK, and creating a ``Credentials`` instance.
 
 .. code:: python
 
     from azure.mgmt.batch import BatchManagementClient, BatchManagementClientConfiguration
+	from azure.common.credentials import UserPassCredentials
 
-    # TODO: Replace this with your subscription id
+    # Replace this with your subscription id
     subscription_id = '33333333-3333-3333-3333-333333333333'
-    # TODO: See above how to get a Credentials instance
-    credentials = ...
+	
+    # See above for details on creating Credentials
+    credentials = UserPassCredentials(
```

Comment:
@annatisch  Ok, I could understand the point. Considering this, can you suggest a good English sentence like "UserPassCredentials are one of the possibility, please see for the other option".
In fact, I'm preparing for the new ADAL which will contains more Credentials (I would like to at least propose a Credentials compatible alias)